src/native/libs/System.Globalization.Native/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+# Required for StaticICULinking
+cmake_minimum_required(VERSION 3.16...3.20)
```

Comment:
What happens when we have > 3.20 installed? Do we want to restrict on max value or can it just be set to 3.16 minimum?

Reply:
you are right.
limiting the max version is probably not what we want to do.
we can set `cmake_minimum_required(VERSION 3.16)`
fixed in the latest commit

Reply:
This is not setting a max version. This is setting the version of CMake to set the policy values for. Setting it to 3.20 ensures that for the dotnet/runtime build, we use a consistent set of CMake policy configurations.